backend/src/util/populateDatabase.js

```diff
@@ -0,0 +1,40 @@
+const fs = require("fs");
```

Comment:
I'm going to rework this function to create a solid database for both testing and development. Probably in my next PR.